src/control/xojfile/LoadHandler.h

```diff
@@ -111,9 +118,13 @@ class LoadHandler
 	ParserPosition pos;
 
 	string creator;
-	int fileversion;
+	int fileVersion;
```

Comment:
You can initialize all vars here, instead of the constructor. But it's not required, only a tip for the future.

Reply:
Just kept how it was previously :wink: